README_WINDOWS.md

```diff
@@ -48,9 +48,11 @@ cd one-click-tangle
     ```console
     ./private-tangle.sh install 30
     ```
-    Interact with private tangle as described in this [README](https://github.com/iotaledger/one-click-tangle/blob/a8ff9269b76fd7f3eb1e4ef95426ca8fc263e52b/hornet-private-net/README.md)
-\
-    b) hornet-mainnet (Chrysalis mainnet node)
+    - Interact with private tangle (start/stop/update) as described in this [README](https://github.com/iotaledger/one-click-tangle/blob/a8ff9269b76fd7f3eb1e4ef95426ca8fc263e52b/hornet-private-net/README.md)
+
+    - Add extra nodes to private tangle as described in this [README](https://github.com/iotaledger/one-click-tangle/blob/chrysalis/hornet-private-net/extra-nodes/README.md)

```

Comment:
@peterokwara I added this line pointing to the readme file for setting up a new node in a private tangle.